plugins/cairo-lang-macro/src/lib.rs

```diff
@@ -1,4 +1,7 @@
 pub use cairo_lang_macro_attributes::*;
+#[doc(hidden)]
+pub use linkme::distributed_slice;
```

Comment:
note for future PR

we might want to move all these hidden items to a wrapping module, like `__priv` to make things more visible